### PR TITLE
Only print container logs when tests fail

### PR DESCRIPTION
**Pull request by tonyreigns**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

################################################################################################

Fixes RTC: 301683

### CODE REVIEW

**Comment by tonyreigns** [comment]
#build

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal/container/fat/TestUtils.java [hunk 1]
```diff
@@ -133,4 +133,28 @@ public static void isContainerStarted(String subString, GenericContainer<?> cont
...
+            Log.info(c, methodName, logs);
+
+        return messageExists;
+    }
+
+    /*
+     * Checks to if the two given log sizes are equal to each other. If not, the container log file will be printed. This was done to reduce the output.txt size.
+     */
+    static boolean compareLogSizes(String methodName, String logs, long expected, long actual) {
+        boolean messageExists = expected == actual;
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal/container/fat/TestUtils.java]:
```suggestion
        boolean messageExists = (expected == actual);
```

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal/container/fat/LoggingServletTest.java [hunk 2]
```diff
@@ -101,17 +100,16 @@ public void testMessageLogs() throws Exception {
...
-        assertTrue("Squence message could not be found.", logs.contains("io.openliberty.sequence: Str"));
-        assertTrue("Log type messagecould not be found.", logs.contains("io.openliberty.type: Str(liberty_message)"));
-        assertTrue("Thread ID message could not be found.", logs.contains("thread.id: Int"));
+                   TestUtils.assertLogContains("testMessageLogs", logs,
+                                               "io.openliberty.module: Str(io.openliberty.microprofile.telemetry.logging.internal.container.fat.MpTelemetryLogApp.MpTelemetryServlet)"));
+        assertTrue("SeverityText message could not be found.", TestUtils.assertLogContains("testMessageLogs", logs, "SeverityText: I"));
+        assertTrue("SeverityNumber message could not be found.", TestUtils.assertLogContains("testMessageLogs", logs, "SeverityNumber: Info"));
+        assertTrue("Squence message could not be found.", TestUtils.assertLogContains("testMessageLogs", logs, "io.openliberty.sequence: Str"));
+        assertTrue("Log type messagecould not be found.", TestUtils.assertLogContains("testMessageLogs", logs, "io.openliberty.type: Str(liberty_message)"));
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal/container/fat/LoggingServletTest.java]:
```suggestion
        assertTrue("Log type message could not be found.", TestUtils.assertLogContains("testMessageLogs", logs, "io.openliberty.type: Str(liberty_message)"));
```

**Review by pgunapal — CHANGES_REQUESTED** [review]

**Review by pgunapal — APPROVED** [review]
Looks good!

**Comment by tonyreigns** [comment]
#libby

**Comment by tonyreigns** [comment]
#libby

**Pull request merged**